README.md

```diff
@@ -1,8 +1,14 @@
 # jdiff
 
-`jdiff` is a lightweight Python library allowing you to examine structured data. `jdiff` provides an interface to intelligently compare JSON data objects and test for the presence (or absence) of keys. You can also examine and compare corresponding key-values.
+`jdiff` is a lightweight Python library allowing you to examine structured data. `jdiff` provides an interface to intelligently compare JSON data objects in a intelligently way.
 
-The library heavily relies on [JMESPath](https://jmespath.org/) for traversing the JSON object and finding the values to be evaluated. More on that [here](#customized-jmespath).
+Our primary use case is the examination of structured data returned from networking devices, such as:
+
+* Compare the operational state of network devices pre and post change
+* Compare operational state of a device vs a "known healthy" state
```

Comment:
```suggestion
* Compare operational state of a device vs a "known healthy" intended state
```

Reply:
I would prefer to not use the word "intended", this will be conflated with configuration state in the same line.